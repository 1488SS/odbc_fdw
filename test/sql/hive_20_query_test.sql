SELECT * FROM hive_test_table;
SELECT * FROM query_hive_test_table;
SELECT * FROM ODBCTablesList('hive_fdw', 1);
SELECT * FROM ODBCTableSize('hive_fdw', 'hive_test_table');
SELECT * FROM ODBCQuerySize('hive_fdw', 'select * from hive_test_table');