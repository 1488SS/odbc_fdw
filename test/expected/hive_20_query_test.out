SELECT * FROM hive_test_table;
 id |   name    | description 
----+-----------+-------------
 1  | test_user | A test user
(1 row)

SELECT * FROM query_hive_test_table;
 EXPR_1 
--------
      1
(1 row)

SELECT * FROM ODBCTablesList('hive_fdw', 1);
 schema  |      name       
---------+-----------------
 default | hive_test_table
(1 row)

SELECT * FROM ODBCTableSize('hive_fdw', 'hive_test_table');
 odbctablesize 
---------------
             1
(1 row)

SELECT * FROM ODBCQuerySize('hive_fdw', 'select * from hive_test_table');
 odbcquerysize 
---------------
             1
(1 row)

